.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -69,3 +69,13 @@ Link related issues below. Insert the issue link or reference after the word "Cl
 - How are we going to debug this?
 - What are the metrics I should take care of?
 - ...
+
+<!-- CI Cheatsheet
+Trigger comments:
+/test             Triggers unit test pipeline
+/test extended    Triggers integration test pipeline
```

Comment:
This should probably include `buildkite test it` and any other buildkite commands that are commonly used.

Reply:
`buildkite test` and `buildkite test extended` do the same

Reply:
Added alternative triggers description